src/main/java/com/aws/greengrass/logmanager/LogManagerService.java

```diff
@@ -89,6 +88,8 @@ public class LogManagerService extends PluginService {
             "lastFileProcessedTimeStamp";
     public static final String PERSISTED_CURRENT_PROCESSING_FILE_LAST_MODIFIED_TIME =
             "currentProcessingFileLastModified";
+    public static final String PERSISTED_CURRENT_PROCESSING_FILE_LAST_ACCESSED_TIME =
+            "currentProcessingFileLastAccessed";
```

Comment:
no reason for this key to be so long, everything takes us space. Just `lastAccessed` is fine.